core/src/epicli/data/common/defaults/configuration/shared-config.yml

```diff
@@ -7,33 +7,5 @@ specification:
   download_directory: /tmp # directory where files and images will be stored just before installing/loading
   vault_location: '' # if empty "BUILD DIRECTORY/vault" will be used
   vault_tmp_file_location: SET_BY_AUTOMATION
```

Comment:
We dont want to have supported OSs configurable anymore?

Reply:
Yes, I've changed this intentionally.